recipe/stdbool.h

```diff
@@ -0,0 +1,3 @@
+typedef int bool;
+#define false 0
+#define true 1
```

Comment:
@jorisvandenbossche and @jorisvandenbossche the compilation on Windows and vc 9 requires msinttypes for `stdint.h` and `stdbool.h`. However, even with those two the compilation fails further down the road and I am not sure if it is worth patching the code or just abandoning vc 9 (that means abandoning Python 2.7).

Reply:
I would find dropping 2.7 support a bit unfortunate, but probably OK to leave it for now to get this moving. 
It also seems the failure is only in the STRTree related code, which is not essential for basic usage (but of course you quickly want to do a spatial join)